src/jsonUtils/computeYogaTree.js

```diff
@@ -29,7 +29,6 @@ const walkTree = (tree: TreeNode, context: Context) => {
 
   return node;
 };
-const treeToNodes = (root: TreeNode, context: Context): yoga.NodeInstance =>
-  walkTree(root, context);
+const treeToNodes = (root: TreeNode, context: Context): yoga.Yoga$Node => walkTree(root, context);
```

Comment:
Nothing wrong here... I have just been trying to figure out what this syntax is: `Yoga$Node`. I have seen it elsewhere in the code within `React$Component` & `React$Element`

Reply:
it's the type definition that comes with yoga: https://github.com/facebook/yoga/blob/master/javascript/sources/entry-common.js#L123

Reply:
Ahhh. So it is just a convention. Awesome.